frontend/app/routes/_public+/intake+/$id+/applicant-information.tsx

```diff
@@ -0,0 +1,144 @@
+    request,
+    params,
+    state: { applicantInformation: parsedDataResult.data },
+  });
+
+  if (['MARRIED', 'COMMONLAW'].includes(parsedDataResult.data.maritalStatus)) {
+    return redirect(`/intake/${id}/spouse-information`, sessionResponseInit);
+  }
+  return redirect(`/intake/${id}/contact-information`, sessionResponseInit);
+}
+
+export default function IntakeFlowApplicationInformation() {
+  const { state, maritalStatuses } = useLoaderData<typeof loader>();
+  const actionData = useActionData<typeof action>();
+  const errorSummaryId = 'error-summary';
+
+  const { i18n, t } = useTranslation(i18nNamespaces);
+
+  /**
+   * Gets an error message based on the provided internationalization (i18n) key.
+   *
+   * @param errorI18nKey - The i18n key for the error message.
+   * @returns The corresponding error message, or undefined if no key is provided.
+   */
+  function getErrorMessage(errorI18nKey?: string): string | undefined {
+    if (!errorI18nKey) return undefined;
+
+    /**
+     * The 'as any' is employed to circumvent typechecking, as the type of
+     * 'errorI18nKey' is a string, and the string literal cannot undergo validation.
+     */
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    return t(`intake:applicant-information.error-message.${errorI18nKey}` as any);
+  }
+
+  const defaultValues = {
+    socialInsuranceNumber: actionData?.formData.socialInsuranceNumber ?? state?.socialInsuranceNumber ?? '',
+    lastName: actionData?.formData.lastName ?? state?.lastName ?? '',
+    maritalStatus: actionData?.formData.maritalStatus ?? state?.maritalStatus ?? '',
+  };
+
+  const errorMessages = {
+    socialInsuranceNumber: getErrorMessage(actionData?.errors.socialInsuranceNumber?._errors[0]),
+    lastName: getErrorMessage(actionData?.errors.lastName?._errors[0]),
+    maritalStatus: getErrorMessage(actionData?.errors.maritalStatus?._errors[0]),
+  };
+
+  const errorSummaryItems = createErrorSummaryItems(errorMessages);
+
+  useEffect(() => {
+    if (actionData?.formData && hasErrors(actionData.formData)) {
+      scrollAndFocusToErrorSummary(errorSummaryId);
+    }
+  }, [actionData]);
+
+  return (
+    <>
+      <p id="form-instructions" className="mb-10">
+        {t('applicant-information.form-instructions')}
+      </p>
+      {errorSummaryItems.length > 0 && <ErrorSummary id={errorSummaryId} errors={errorSummaryItems} />}
+      <Form method="post" aria-describedby="form-instructions" noValidate>
+        <InputField
+          id="socialInsuranceNumber"
+          name="socialInsuranceNumber"
+          label={t('applicant-information.sin')}
+          required
+          inputMode="numeric"
+          pattern="\d{9}"
+          minLength={9}
+          maxLength={9}
+          defaultValue={defaultValues.socialInsuranceNumber}
+          errorMessage={errorMessages.socialInsuranceNumber}
+        />
+        <div className="grid gap-3 md:grid-cols-4">
```

Comment:
Add `max-w-prose` on the `form` to limit the form width.

```suggestion
        <div className="grid gap-3 md:grid-cols-2">
```